bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/internal/RuleRegistryImpl.java

```diff
@@ -504,8 +504,8 @@ public void added(Provider<Rule> provider, Rule element) {
             String rUID = ruleEngine.getUniqueId();
             ruleWithUID = initRuleId(rUID, element);
         }
-        super.added(provider, ruleWithUID);
```

Comment:
> BTW wdyt about removing redundant NOT_INITIALIZED event.

You are referring to [this suggetion](https://github.com/eclipse/smarthome/issues/2823), right? If so, please see my comment on that issue.